package.json

```diff
@@ -11,9 +11,12 @@
   "devDependencies": {
     "@commitlint/cli": "^8.1.0",
     "@commitlint/config-lerna-scopes": "^8.1.0",
+    "eslint": "^5.16.0",
```

Comment:
I'd disable the `^`. In the other repos we pine deps.

Reply:
I prefer to not pin all the dependencies as some of them are better to allow automatic update. I like the caret because we can get minor updates with new features and improves, but we can set them with tilde to get only patches. I will merge this PR but tomorrow I'd like to talk about this to get a consensus :smiley:

Reply:
Minor updates are dangerous, let me give you a 2 close examples that happened to Verdaccio lately.

- `request@2.87.0`: https://github.com/request/request/blob/master/CHANGELOG.md#v2880-20180810 they have introduced I think in this https://github.com/request/request/issues/3038 the next minor version completely breaks our functional testing at Verdaccio, it took me 4 days to realize the issue was within the library caused by a sub dependency.
- `express@4.18.0`: If you try to look for such version at npmjs you will realise the latest is `4.17.1`, 🤔 but why at Verdaccio registry the `express@4.18.0` is available? https://registry.verdaccio.org/express 

Let's see the time field.
```
"4.17.0": "2019-05-17T01:57:40.690Z",
    "4.18.0": "2019-05-18T14:39:53.454Z",
    "4.17.1": "2019-05-26T04:25:34.606Z"
```

So, `4.18.0` was shipped and removed almost instantly, but in that gap of time I upgraded (lucky me) and the `4.18.0` remains cached at Verdaccio registry. The `^` upgraded safely such version and yarn was failing at Docker for some reason, took me a while to realise the `4.18.0` was removed ant he latest was `4.17.1`.

My point is, **semver** works with human capabilities, we fail, deploys might goes wrong. For such reason I prefer manual upgrade, so I know exactly whether an upgrade might blow up the whole build.

Reply:
We have decided to fix the version of dependencies and keep caret (`^`) on all devDependencies for a while. If we find troubles in a future with devDependencies, we will try to set them to tilde (`~`) to get, at least, patches. In the worst case, they will be fixed as dependencies :smiley: